src/libraries/skinDynLib/test/skinContactTest.cpp

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2017 Fondazione Istituto Italiano di Tecnologia (IIT)
+ * CopyPolicy: Released under the terms of the GNU GPL v2.0.
+ */
+
+#include <cassert>
+#include <cmath>
+#include <ctime>
```

Comment:
Now I understand what you mean. 
Well, it depends then on what you want to measure. If you want to measure the CPU time, that you have to stick to the old clock library.